csrc/scheduler/registry.cpp

```diff
@@ -56,24 +57,7 @@ SchedulerRuntimeInfo::SchedulerRuntimeInfo(
       NVF_ERROR(alloc_sizes.size() == alloc_strides.size());
 
       input_ptrs_[fusion_inp] = (size_t)(metadata->*&TensorMetaData::data);
-
-      // find and push discontiguous stride
-      int64_t dtype_size = dataTypeSize(input_tv->dtype());
-      input_discontig_strides_[fusion_inp] = {};
-      int64_t dims = (int64_t)alloc_strides.size();
-      int64_t expected_stride = 1;
-      for (int64_t dim = dims - 1; dim >= 0; dim--) {
-        auto size = alloc_sizes.at(dim);
-        if (size <= 1) {
-          continue;
-        }
-        auto stride = alloc_strides.at(dim);
-        if (stride != expected_stride) {
-          input_discontig_strides_[fusion_inp].push_back(stride * dtype_size);
-          expected_stride = stride;
-        }
-        expected_stride *= size;
-      }
+      input_strides_elts_[fusion_inp] = alloc_strides.vec();
```

Comment:
I think for pointwise fusion, we still want to use `input_discontig_strides_`, because for a float tensor of shape `(102400000, 3)`, we want to vectorize 4. But this change will make it un-vectorizable?

Reply:
You're right. We need to preserve that and also record the unmodified stride for use on its own.

Reply:
Pushed a less invasive change that preserves previous behavior but adds a new stride member and accessor.